python/tests/test_balance_metrics.py

```diff
@@ -121,8 +135,37 @@ def tree(self):
     def test_sackin(self):
         assert self.tree().sackin_index() == 14
 
+    def test_colless(self):
+        assert self.tree().colless_index() == 6
+
 
-class TestMultiRoot:
+class TestMultiRootBinary:
+    # 3.00┊            15     ┊
+    #     ┊          ┏━━┻━┓   ┊
+    # 2.00┊   11     ┃   14   ┊
+    #     ┊  ┏━┻━┓   ┃  ┏━┻┓  ┊
+    # 1.00┊  9  10  12  ┃ 13  ┊
+    #     ┊ ┏┻┓ ┏┻┓ ┏┻┓ ┃ ┏┻┓ ┊
+    # 0.00┊ 0 1 2 3 4 5 6 7 8 ┊
+    #     0                   1
+    @tests.cached_example
```

Comment:
I added this test because the only multi roots test was ternary, thus expecting `math.inf` for Colless index. I wanted to add a multi root test that outputs a real value. @jeromekelleher

Reply:
We can probably get rid of the multiroot ternary example, it's not checking anything we're not doing elsewhere.

Reply:
Yes, just removed it.